test/help/wasm-wast.txt

```diff
@@ -10,14 +10,15 @@ examples:
   # parse binary file test.wasm and write s-expression file test.wast
   $ wasm2wast test.wasm -o test.wast
 
-  # parse test.wasm and write test.wast, using the debug names, if any
-  $ wasm2wast test.wasm --debug-names -o test.wast
+  # parse test.wasm, write test.wast, but ignore the debug names, if any
+  $ wasm2wast test.wasm --no-debug-names -o test.wast
 
 options:
   -v, --verbose                   use multiple times for more info
   -h, --help                      print this help message
   -o, --output=FILENAME           output file for the generated wast file, by default use stdout
       --use-libc-allocator        use malloc, free, etc. instead of stack allocator
-      --debug-names               Read debug names from the binary file
+      --debug-names               Read debug names from the binary file (default)
```

Comment:
Why not just drop this option.  I can't see how it could be useful if its already the default.